test-case/multiple-pipeline.sh

```diff
@@ -8,14 +8,15 @@
 ## Description:
 ##    Run multiple pipelines in parallel
 ##    Rule:
-##      a. playback first mode: playback pipelines will be started first, if playback
+##      1. playback first mode: playback pipelines will be started first, if playback
 ##         pipeline count is less than requested pipeline count defined by -c option,
 ##         capture pipelines will be started to ensure requested number of pipelines are
 ##         running in parallel.
-##      b. capture first mode: same with playback mode, but capture pipelines will be
+##      2. capture first mode: same with playback mode, but capture pipelines will be
 ##         started first.
-##      c. all pipelines mode: if requested pipeline count is greater than the number of
-##         pipelines defined in topology, all pipelines will be started.
+##      3. all pipelines mode: run all pipelines in parallel by
+##          3.a use option '-f a' OR
```

Comment:
```
a.
b. 
```
will looks better. You already have the intend.

Reply:
I think we should align on this, how about
level 1: use number, 1.
level 2: use lower case character,  a.
level 3: use upper case character, A. ???